src/Stratis.Bitcoin.Tests/NodeConfiguration/NodeSettingsTest.cs

```diff
@@ -148,5 +151,48 @@ public void NodeSettings_CanReadMultiValueFromNone()
             // Assert
             Assert.Empty(result);
         }
+
+        /// <summary>
+        /// Verifies API port value can be passed in on startup.
+        /// </summary>
+        [Fact]
+        public void NodeSettings_CanOverrideOnlyApiPort()
+        {
+            const int apiport = 12345;
+            var network = Networks.Networks.Bitcoin;
+            var nodeSettings = new NodeSettings(networksSelector: network, args: new[] {  $"-apiport={apiport}" });
```

Comment:
@thefazzer we wanna possibly move away from network selector at some point, so can you perhaps just pass in `new BitcoinRegTest()` here? The same with the test below.